app/services/tasks.py

```diff
@@ -28,8 +29,20 @@ def update_task(data ,id):
         task.title = data["title"]
     if "description" in data:
         task.description = data["description"] 
+    if "status" in data:
+        task.status = data["status"]
     db.session.commit()
 
 def delete_task(task):
     db.session.delete(task)
+    db.session.commit()
+
+def get_tasks_assign(user_id):
+    tasks = Tasks.query.all()
+    task_list = [task for task in tasks if task.assigned_to == user_id ]
```

Comment:
We can query all the needed tasks only, No need for finding it by looping over all the tasks.